.github/workflows/release.yml

```diff
@@ -53,3 +53,5 @@ jobs:
           generateReleaseNotes: true
           allowUpdates: true
           artifactErrorsFailBuild: true
+          omitBodyDuringUpdate: true
+          omitNameDuringUpdate: true
```

Comment:
That's an interesting option. How is it even supposed to know which release to update if the name doesn't match? I'm not really expecting an answer here, just thinking outloud.

Reply:
I think it uses the tag name to figure out which release to use.